WebView/app/src/main/java/com/android/samples/webviewdemo/MainActivity.kt

```diff
@@ -19,22 +19,23 @@ package com.android.samples.webviewdemo
 import android.content.Context
 import android.content.Intent
 import android.content.pm.ApplicationInfo
+import android.net.Uri
 import android.os.Build
 import android.os.Bundle
+import android.util.Log
 import android.webkit.JavascriptInterface
 import android.webkit.WebResourceRequest
 import android.webkit.WebResourceResponse
 import android.webkit.WebView
-import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
 import androidx.core.content.ContextCompat.startActivity
-import androidx.webkit.WebViewAssetLoader
-import androidx.webkit.WebViewClientCompat
+import androidx.webkit.*
 import com.android.samples.webviewdemo.databinding.ActivityMainBinding
+import kotlinx.android.synthetic.main.activity_main.*
```

Comment:
This shouldn't be needed, since we're using view binding.

Reply:
Ok